### ISSUE TITLE
Python bindings for `amgcl`

### ISSUE THREAD

**Issue by ma-sadeghi** [ddemidov/amgcl#233]
Just came across this amazing repo, I was just wondering if there's any working Python interface for `amgcl`. Thank you so much!

**Comment by ddemidov** [comment]
Yes, there is a wrapper code using pybind11 in the [pyamgcl](https://github.com/ddemidov/amgcl/tree/master/pyamgcl) folder. Also see these examples:

- [examples/pysolver.py](https://github.com/ddemidov/amgcl/blob/master/examples/pysolver.py)
- [examples/pyprecond.py](https://github.com/ddemidov/amgcl/blob/master/examples/pyprecond.py)

**Comment by tvercaut** [comment]
Apologies for reviving this old issue. Are there any plans to get a packaged version on pypi? This would be a nice feature to help quickly try the available wrapped solvers.

**Comment by ddemidov** [comment]
There are no such plans currently, but I would accept a PR for this :). Note that the python package here is more of an example, as it does not cover all of the possible functionality.
